Makefile

```diff
@@ -1,6 +1,39 @@
-develop: clean
-	docker-compose up --build
+.PHONY: clean install run test deploy docker-run docker-test docker-bash
+
+DOCKER_COMMAND=docker run -it --rm --user=`id -u`:`id -g` -v $$PWD:/srv/jekyll kongdocs
```

Comment:
If I remove the ```--user=`id -u`:`id -g``` flag, I'm able to run `make docker-run`. Otherwise, I get permissions errors.

Reply:
that will fix it for `make docker-run` but it leaves files on your OS that are owned by root. So if you
```
make docker-run
make run
```
it's full of sadness

A possible solution would be to configure npm to install `node_modules` somewhere not shared with the host OS